features/step_definitions.rb

```diff
@@ -1,12 +1,13 @@
 Before do
+  FileUtils.rm_rf(Paths.test_dir) if Paths.test_dir.exist?
   FileUtils.mkdir_p(Paths.test_dir) unless Paths.test_dir.directory?
   Dir.chdir(Paths.test_dir)
 end
 
 #
 
 After do
-  Paths.test_dir.rmtree if Paths.test_dir.exist?
```

Comment:
Out of curiosity, why does `.rmtree` fail? Can you paste the error or describe the issue? It should be identical to `FileUtils.rm_rf` IIRC.